userspace/libscap/engine/gvisor/scap_gvisor.cpp

```diff
@@ -42,7 +40,7 @@ constexpr uint32_t max_ready_sandboxes = 32;
 constexpr size_t max_message_size = 300 * 1024;
 constexpr size_t initial_event_buffer_size = 32;
 constexpr int listen_backlog_size = 128;
-constexpr size_t max_line_size = 2048;
+constexpr size_t max_line_size = 8192;
```

Comment:
This was bumped but i cannot see it used in the diff. Is it used somewhere else?
Perhaps inside `runsc()` method?

Reply:
Yes, it is used in the `runsc()` method. I decided to increase it because json strings containing proc information could potentially be longer than the output strings we dealt with until now!